library/include/modules/Units.h

```diff
@@ -62,213 +63,270 @@ namespace df
-DFHACK_EXPORT std::string getRaceChildName(df::unit* unit);
-DFHACK_EXPORT std::string getReadableName(df::unit* unit);
-
-DFHACK_EXPORT double getAge(df::unit *unit, bool true_age = false);
-DFHACK_EXPORT int getKillCount(df::unit *unit);
-
-DFHACK_EXPORT int getNominalSkill(df::unit *unit, df::job_skill skill_id, bool use_rust = false);
-DFHACK_EXPORT int getEffectiveSkill(df::unit *unit, df::job_skill skill_id);
-DFHACK_EXPORT int getExperience(df::unit *unit, df::job_skill skill_id, bool total = false);
-
-DFHACK_EXPORT bool isValidLabor(df::unit *unit, df::unit_labor labor);
-DFHACK_EXPORT bool setLaborValidity(df::unit_labor labor, bool isValid);
-
-DFHACK_EXPORT int computeMovementSpeed(df::unit *unit);
-DFHACK_EXPORT float computeSlowdownFactor(df::unit *unit);
-
-struct NoblePosition {
-    df::historical_entity *entity;
-    df::entity_position_assignment *assignment;
-    df::entity_position *position;
-};
-
-DFHACK_EXPORT bool getNoblePositions(std::vector<NoblePosition> *pvec, df::unit *unit);
-
-DFHACK_EXPORT std::string getProfessionName(df::unit *unit, bool ignore_noble = false, bool plural = false);
-DFHACK_EXPORT std::string getCasteProfessionName(int race, int caste, df::profession pid, bool plural = false);
-
-DFHACK_EXPORT int8_t getProfessionColor(df::unit *unit, bool ignore_noble = false);
-DFHACK_EXPORT int8_t getCasteProfessionColor(int race, int caste, df::profession pid);
-
-DFHACK_EXPORT df::goal_type getGoalType(df::unit *unit, size_t goalIndex = 0);
-DFHACK_EXPORT std::string getGoalName(df::unit *unit, size_t goalIndex = 0);
-DFHACK_EXPORT bool isGoalAchieved(df::unit *unit, size_t goalIndex = 0);
-
-DFHACK_EXPORT df::activity_entry *getMainSocialActivity(df::unit *unit);
-DFHACK_EXPORT df::activity_event *getMainSocialEvent(df::unit *unit);
-
-// stress categories - 0 is highest stress
-DFHACK_EXPORT extern const std::vector<int32_t> stress_cutoffs;
-DFHACK_EXPORT int getStressCategory(df::unit *unit);
-DFHACK_EXPORT int getStressCategoryRaw(int32_t stress_level);
-
-DFHACK_EXPORT void subtractActionTimers(color_ostream &out, df::unit *unit, int32_t amount, df::unit_action_type affectedActionType);
-DFHACK_EXPORT void subtractGroupActionTimers(color_ostream &out, df::unit *unit, int32_t amount, df::unit_action_type_group affectedActionTypeGroup);
-DFHACK_EXPORT void multiplyActionTimers(color_ostream &out, df::unit *unit, float amount, df::unit_action_type affectedActionType);
-DFHACK_EXPORT void multiplyGroupActionTimers(color_ostream &out, df::unit *unit, float amount, df::unit_action_type_group affectedActionTypeGroup);
-DFHACK_EXPORT void setActionTimers(color_ostream &out, df::unit *unit, int32_t amount, df::unit_action_type affectedActionType);
-DFHACK_EXPORT void setGroupActionTimers(color_ostream &out, df::unit *unit, int32_t amount, df::unit_action_type_group affectedActionTypeGroup);
-
-}
+namespace DFHack {
+    namespace Units {
+        /**
```

Comment:
I think this adds indentation for insufficient semantic clarity. We do not normally indent namespace blocks (unless they are very short), and I think this is better left without indentation.

Reply:
Should I unindent Maps.h or leave it as it is?

Reply:
yeah, for consistency's sake